quokka/core/content/utils.py

```diff
@@ -10,9 +10,10 @@ def url_for_content(content, include_ext=True):
     else:
         data = content
 
+    category_slug_data = data.get('category_slug')
+    category_data = slugify_category(data.get('category') or '')
```

Comment:
See: data.get('category', '') maybe it's simpler

Reply:
done;
commit => https://github.com/marcosptf/quokka/commit/9ab02456bf6de5a54e02ff47be34ba504bd33a5d